tools/send-to-caas/send-utils.js

```diff
@@ -306,7 +306,7 @@ const getBulkPublishLangAttr = async (options) => {
 };
 
 const getCountryAndLang = async (options) => {
-  const langStr = window.location.pathname === '/tools/send-to-caas/bulkpublisher.html'
+  const langStr = window.location.pathname.includes('/tools/send-to-caas/bulkpublisher.')
```

Comment:
Should you also remove the `.`?

Reply:
I left it intentionally :) to match bulkpublisher.html and bulkpublisher.beta.html